aten/src/ATen/SparseCsrTensorImpl.cpp

```diff
@@ -150,14 +150,22 @@ void SparseCsrTensorImpl::resize_as_sparse_csr_tensor_(const Tensor& src) {
       !has_symbolic_sizes_strides_,
       "resize_as_sparse_csr_tensor_ called on tensor with symbolic shape");
   set_layout(src.layout());
+  Tensor compressed_indices;
+  Tensor plain_indices;
+  std::tie(compressed_indices, plain_indices) =
+      sparse_csr::getCompressedPlainIndices(src);
+
+  // compressed indices are always crow_indices_ on the impl
   crow_indices_ = at::empty_like(
```

Comment:
```suggestion
  crow_indices_ = compressed_indices.clone();
```
would make more sense as it guarantees the validity of sparse compressed tensor invariants.

Reply:
I agree, only `clone`. `empty_like` on indices does not make sense.